torch/csrc/jit/passes/subgraph_rewrite.cpp

```diff
@@ -17,13 +43,16 @@ graph(%x, %w, %b):
       R"IR(
 graph(%x, %w, %b):
   %r = aten::convrelu(%x, %w, %b)
-  return (%r))IR");
+  return (%r))IR",
```

Comment:
Do we use the default patterns anywhere btw? I added them in the beginning, but I suspect we register patterns explicitly in all use cases - if that's the case we could probably nuke this.